### ISSUE TITLE
[Bug]: <Error loading the game in iOS 12.5>

### ISSUE THREAD

**Issue by pratoputo** [MelvorIdle/melvoridle.github.io#1688]
### Describe the bug

I'm using an iPhone 6 with the software version 12.5.5 (Apple doesn't update iPhone 6 anymore) and when I try to load my most recent save I get the error: 

populateMasteryObject@https://ios.melvoridle.com/assets/js/game/mastery2.js?1297:4:31
updateWindow@https://ios.melvoridle.com/assets/js/game/main.js?1297:4:264
finaliseLoad@https://ios.melvoridle.com/assets/js/game/main.js?1297:372:13
selectCharacter@https://ios.melvoridle.com/assets/js/game/main.js?1297:363:185
onloadEvent@https://ios.melvoridle.com/assets/js/game/save.js?1297:61:232
https://ios.melvoridle.com/assets/js/game/save.js?1297:66:382
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84826
each@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:2971
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84807
r@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28088
fireWith@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28852
c@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:79495
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:81938"

It's very common for the game to bug after an update, as I usually reach Malcs to adress them, probably due to the iOS outdatted version.

I would like to point that the error pops both in app and in Firefox or Safari.

### Screenshots

![imagem](https://user-images.githubusercontent.com/94323937/141700002-53885b59-3ed5-44fd-8f7e-808688eab86b.png)

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [X] Firefox
- [ ] Edge
- [X] Safari
- [X] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

Beta v1.0

### Game Subversion

?1297

### Are you using any scripts or extensions?

No

**Comment by MelvorIdle** [comment]
Is this an old save that you were trying to load?

**Comment by pratoputo** [comment]
No, I exported a save in my PC and pasted on the phone. After the error pops, it appears a button to "fix my save", but it doesn't work either.

**Comment by MelvorIdle** [comment]
Do you have a cloud account? Let me know the username and I will take a look at the save and try to fix it

**Comment by pratoputo** [comment]
Yes I do, my username is pratoputo

**Comment by pratoputo** [comment]
The problem remains, any fix?

**Comment by pratoputo** [comment]
Still can't play the game

**Comment by MelvorIdle** [comment]
Here, try importing this into the game: 
[melvoridlesave_pratoputo.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/7681400/melvoridlesave_pratoputo.txt)

**Comment by pratoputo** [comment]
Same thing, I belive the problem is that my OS is outdated compared to the version of the app in the store. In App Store, it shows to update the game, but it pops a error saying my OS is outdated and the version available is for iOS 13+

**Comment by MelvorIdle** [comment]
It shouldn't relate to that because we still have iOS 12 users players playing.

Are you able to delete every character and player the game? (I have your cloud saves on file as backup so don't worry about those)

**Comment by pratoputo** [comment]
I already tried that, creating a new character in different game modes, but it pops the same error

**Comment by MelvorIdle** [comment]
Okay, I might have a fix for you. Give me 5 minutes and I'll let you know when its live

**Comment by MelvorIdle** [comment]
Fix is now live. Can you load the game now? (Force restart the app)

**Comment by pratoputo** [comment]
Sadly it seems it didn't work

**Comment by pratoputo** [comment]
The error is different now too

populateMasteryObject@https://ios.melvoridle.com/assets/js/game/mastery2.js?1363:4:31
updateWindow@https://ios.melvoridle.com/assets/js/game/main.js?1363:4:264
finaliseLoad@https://ios.melvoridle.com/assets/js/game/main.js?1363:374:13
selectCharacter@https://ios.melvoridle.com/assets/js/game/main.js?1363:366:240
onloadEvent@https://ios.melvoridle.com/assets/js/game/save.js?1363:62:232
https://ios.melvoridle.com/assets/js/game/save.js?1363:66:614
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84826
each@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:2971
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:84807
r@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28088
fireWith@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:28852
c@https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:79495
https://cdn.melvor.net/core/v018/assets/js/oneui.core.min.js:6:81938
